### DISCUSSION TITLE
Where can I find tutorial on Polygon counting using Supervision?

### DISCUSSION THREAD

**Discussion by yeongnamtan** [roboflow/supervision#599] [Q&A]
Where can I find tutorial on Polygon counting using Supervision?

**Comment by SkalskiP** [comment]
Hi @yeongnamtan! 👋🏻 Thanks for your interest in supervision. Take a look here:
- https://youtu.be/l_kf9CfZ_8M
- https://colab.research.google.com/github/roboflow-ai/notebooks/blob/main/notebooks/how-to-detect-and-count-objects-in-polygon-zone.ipynb
